src/v/kafka/server/handlers/list_groups.cc

```diff
@@ -30,12 +30,23 @@ ss::future<response_ptr> list_groups_handler::handle(
     request.decode(ctx.reader(), ctx.header().version);
     klog.trace("Handling request {}", request);
 
-    auto groups = co_await ctx.groups().list_groups();
-    // group listing is still returned even if some partitions are
-    // still in the process of loading/recovering.
+    auto&& [error, groups] = co_await ctx.groups().list_groups();
```

Comment:
I don't think so. what do you have in mind?

Reply:
`auto` instead of `auto&&`

Reply:
my understand is that `auto&&` will bind to the return value temporary instead of building a hidden variable for the return value. is that not the case?

Reply:
In this case I think it's the same thing.